designer/submodules/packages/components/src/components/usages/filtersPart.tsx

```diff
@@ -20,14 +20,16 @@ export function FiltersPart({
     const { t } = useTranslation();
     const { getFilter, setFilter } = useFilterContext<UsagesFiltersModel>();
 
-    const statusFilters: Array<keyof UsagesFiltersModel> = ["HIDE_DEPLOYED", "HIDE_NOT_DEPLOYED"];
     const otherFilters: Array<keyof UsagesFiltersModel> = ["HIDE_SCENARIOS", "HIDE_FRAGMENTS"];
 
     return (
         <QuickFilter<UsagesFiltersModel> isLoading={isLoading} filter="TEXT">
             <Stack direction="row" spacing={1} p={1} alignItems="center" divider={<Divider orientation="vertical" flexItem />}>
-                <FilterMenu label={t("table.filter.STATUS", "Status")} count={statusFilters.filter((k) => getFilter(k)).length}>
-                    <StatusOptionsStack />
+                <FilterMenu label={t("table.filter.STATUS", "Status")} count={getFilter("STATUS", true).length}>
+                    <StatusOptionsStack
+                        options={filterableValues["status"]}
+                        withArchived={false}
```

Comment:
Recently we've replaced filter-by-inversion approach to filtering by status or filtering by type (e.g. HIDE_NOT_DEPLOYED replaced by DEPLOYED). It was changed in Scenarios tab but old approach remained in Components -> Usages tab.

Reply:
It's a bug. That switch is not handled in Usages (see componentView.tsx) because Components refers only to not-archived scenarios or fragments. Hence in componentView you can find only: hide_scenarios, hide_fragments, hide_deployed and hide_not_deployed, created_by and category. I guess the switch appeared here not intentionally because it uses the same component `FiltersPart` as Scenarios tab.

Reply:
Thats why I've added "withArchived" flag to `FiltersPart` compoennt.